### PR TITLE
Change Payment to Contact

### PR DESCRIPTION
**Pull request by cjharkins**

### CODE REVIEW

File: src/forms/EditListingContactForm/EditListingContactForm.js [hunk 1]
```diff
@@ -34,38 +34,24 @@ export const EditListingPaymentFormComponent = (props) => {
...
-              id="payPalEmail"
-              name="payPalEmail"
-              className={css.title}
-              type="text"
-              label={questionATitle}
-              placeholder={questionATitlePlaceholderMessage}
-              autoFocus
-            />
-
             <FieldTextInput
```
Comment (masonembry) [inline src/forms/EditListingContactForm/EditListingContactForm.js:55]: Can we change this to type `tel` or whatever it is? https://www.w3schools.com/tags/att_input_type_tel.asp. A pattern to validate would be great too.

File: src/components/EditListingWizard/EditListingWizard.js [hunk 2]
```diff
@@ -116,9 +116,9 @@ const tabCompleted = (tab, listing) => {
     case ABOUTTHISSERVICE:
       const { averageTurnAroundTime, explainMore } = publicData
       return !!averageTurnAroundTime && !!explainMore
-    case PAYMENT:
-      const { payPalEmail, phoneNumber } = publicData
-      return !!payPalEmail && !!phoneNumber
+    case CONTACT:
+      const { phoneNumber } = publicData
+      return !!phoneNumber
```
Comment (masonembry) [inline src/components/EditListingWizard/EditListingWizard.js:121]: Can we validate that this is a real phone number before we let them move past that form? I don't care where we do it, here or in the input itself.

**Review by masonembry — APPROVED** [review]
Please see the comments first before merging.

**Review by embryCODE — APPROVED** [review]

File: src/forms/EditListingContactForm/EditListingContactForm.js (hunk 1 above)
Comment (cjharkins) [reply src/forms/EditListingContactForm/EditListingContactForm.js:55]: I have the next PR that addresses this, you can't add type="tel" to this component, however, Flex gives you a Text Phone component that does what you are asking for.

File: src/components/EditListingWizard/EditListingWizard.js (hunk 2 above)
Comment (cjharkins) [reply src/components/EditListingWizard/EditListingWizard.js:121]: It will be in next PR

**Pull request merged**